### PR TITLE
feat: add chart prices to SDK

### PR DESCRIPTION
**Pull request by 0xKoaj**

### CODE REVIEW

File: src/services/prices/price-sources/aggregator-price-source.ts [hunk 1]
```diff
@@ -80,6 +80,45 @@ export class AggregatorPriceSource implements IPriceSource {
...
+        }),
+      timeout: config?.timeout,
+    });
+    // TODO: Support more than one source
+    return Object.fromEntries(
+      Object.entries(collected).map(([chain, tokens]) => [
+        parseInt(chain),
+        Object.fromEntries(Object.entries(tokens).map(([token, prices]) => [token, prices[0]])),
+      ])
+    );
```
Comment (0xsambugs) [inline src/services/prices/price-sources/aggregator-price-source.ts:119]: May I suggest something a little bit simpler, like this?
```suggestion
    // TODO: Support more than one source
    const sourcesInChains = getSourcesThatSupportRequestOrFail(tokens, this.sources, 'getChart');
    return sourcesInChains[0].getChart({ tokens, span, period, bound, searchWidth, config }) ?? {}
```

File: src/shared/defi-llama.ts [hunk 2]
```diff
@@ -176,7 +239,7 @@ export class DefiLlamaClient {
       '?' +
       Object.entries(extraParams)
         .map(([key, value]) => `${key}=${value}`)
-        .join('&=');
+        .join('&');
```
Comment (0xsambugs) [inline src/shared/defi-llama.ts:242]: Good catch!

File: src/shared/defi-llama.ts [hunk 3]
```diff
@@ -190,6 +253,27 @@ export class DefiLlamaClient {
...
+        const response = await this.fetch.fetch(url, { timeout: config?.timeout });
+        const { coins }: { coins: Record<TokenId, FetchPricesResult> } = await response.json();
+        return coins;
+      } catch {
+        throw new Error('Request to Defi Llama API failed');
+      }
+    });
+    const responses = await Promise.all(requests);
+    return responses.reduce((accum, curr) => ({ ...accum, ...curr }), {});
+  }
```
Comment (0xsambugs) [inline src/shared/defi-llama.ts:276]: I think this isn't being used
```suggestion
```

**Review by 0xsambugs — APPROVED** [review]
Great work!
Also, please ignore the failing integration test, I'll take a look

File: src/services/prices/price-sources/aggregator-price-source.ts (hunk 1 above)
Comment (0xKoaj) [reply src/services/prices/price-sources/aggregator-price-source.ts:119]: Making things complicated is my middle name

**Pull request merged**